Missing freetype include in pngwriter.h

For building PIConGPU `0.4.0-rc1`, `pic-build -b "omp2b"` on Titan (ORNL) using pngwriter with own build freetype2 I need to add the following include line to `pngwriter.h`:
`#include <freetype/ftimage.h>`

Looks like that in `pngwriter.h` context:
```C++
// REMEMBER TO ADD -DNO_FREETYPE TO YOUR COMPILATION FLAGS IF PNGwriter WAS
// COMPILED WITHOUT FREETYPE SUPPORT!!!
//
// <png.h> must be included before FreeType headers.
#ifndef NO_FREETYPE
#include <ft2build.h>
#include <freetype/ftimage.h>
#include FT_FREETYPE_H
#endif
```

Below is the build and makefile output.
```bash
$ export VERBOSE=ON; pic-build -b "omp2b"
build directory: .build
cmake command: SCOREP_WRAPPER=OFF cmake  -DCMAKE_INSTALL_PREFIX=.. -DPIC_EXTENSION_PATH=..   -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE=ON /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu
Re-run cmake no build system arguments
-- The C compiler identification is GNU 4.9.3
-- The CXX compiler identification is GNU 4.9.3
-- Cray Programming Environment 2.5.13 C
-- Check for working C compiler: /opt/cray/craype/2.5.13/bin/cc
-- Check for working C compiler: /opt/cray/craype/2.5.13/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Cray Programming Environment 2.5.13 CXX
-- Check for working CXX compiler: /opt/cray/craype/2.5.13/bin/CC
-- Check for working CXX compiler: /opt/cray/craype/2.5.13/bin/CC -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenMP_C: -fopenmp (found version "4.0")
-- Found OpenMP_CXX: -fopenmp (found version "4.0")
-- Found OpenMP: TRUE (found version "4.0")
-- ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED
-- Found alpaka: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include (found version "0.3.3")
-- _cupla_COMPILE_OPTIONS_PUBLIC: -fopenmp;-std=c++11;-ftemplate-depth-512;-Wall;-Wextra;-Wno-unknown-pragmas;-Wno-unused-parameter;-Wno-unused-local-typedefs;-Wno-attributes;-Wno-reorder;-Wno-sign-compare
-- _cupla_COMPILE_DEFINITIONS_PUBLIC: ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED;ALPAKA_DEBUG=0;CUPLA_STREAM_ASYNC_ENABLED=1
-- _cupla_INCLUDE_DIRECTORIES_PUBLIC: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include;/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include
-- _cupla_LINK_LIBRARIES_PUBLIC: -fopenmp;Boost::boost;general;pthread;general;rt
-- Found cupla: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include (found version "0.1.0")
-- Found MPI_C: /opt/cray/craype/2.5.13/bin/cc (found version "3.1")
-- Found MPI_CXX: /opt/cray/craype/2.5.13/bin/CC (found version "3.1")
-- Found MPI: TRUE (found version "3.1")
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   filesystem
--   system
--   math_tr1
-- Boost: result_of with TR1 style and decltype fallback
-- Boost: Disable template aliases
-- Found OpenMP_C: -fopenmp (found version "4.0")
-- Found OpenMP_CXX: -fopenmp (found version "4.0")
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE
-- Found OpenMP_C: -fopenmp (found version "4.0")
-- Found OpenMP_CXX: -fopenmp (found version "4.0")
-- Found ZLIB: /usr/lib64/libz.so (found version "1.2.7")
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   program_options
--   regex
--   filesystem
--   system
--   math_tr1
--   serialization
-- Disable 'cuda_memtest' build: not supported by clang or non CUDA alpaka backends
-- Can NOT find 'adios_config' - set ADIOS_ROOT or check your PATH
-- Could NOT find ADIOS (missing: ADIOS_LIBRARIES ADIOS_INCLUDE_DIRS) (Required is at least version "1.13.1")
-- HDF5: Using hdf5 compiler wrapper for all C compiling
-- Found HDF5: Included by compiler wrappers (found version "1.8.16")
-- Found Splash: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/lib/cmake/Splash
-- Found PNG: /usr/lib64/libpng.so (found version "1.2.31")
-- Found Freetype: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/lib64/libfreetype.a (found version "2.9.1")
-- Found PNGwriter: /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/lib/cmake/PNGwriter
-- Could NOT find ISAAC - set ISAAC_DIR or check your CMAKE_PREFIX_PATH
-- Configuring done
-- Generating done
-- Build files have been written to: /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -H/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu -B/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build --check-build-system CMakeFiles/Makefile.cmake 0
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_progress_start /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[1]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make -f CMakeFiles/picongpu-hostonly.dir/build.make CMakeFiles/picongpu-hostonly.dir/depend
make -f CMakeFiles/cupla.dir/build.make CMakeFiles/cupla.dir/depend
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
cd /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build && /ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_depends "Unix Makefiles" /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu-hostonly.dir/DependInfo.cmake --color=
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
cd /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build && /ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_depends "Unix Makefiles" /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/cupla.dir/DependInfo.cmake --color=
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu-hostonly.dir/DependInfo.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu-hostonly.dir/depend.internal".
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu-hostonly.dir/depend.internal".
Scanning dependencies of target picongpu-hostonly
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/cupla.dir/DependInfo.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/cupla.dir/depend.internal".
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/cupla.dir/depend.internal".
Scanning dependencies of target cupla
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make -f CMakeFiles/cupla.dir/build.make CMakeFiles/cupla.dir/build
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
[  5%] Linking CXX static library libcupla.a
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -P CMakeFiles/cupla.dir/cmake_clean_target.cmake
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_link_script CMakeFiles/cupla.dir/link.txt --verbose=ON
/usr/bin/ar qc libcupla.a
/usr/bin/ranlib libcupla.a
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
[  5%] Built target cupla
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make -f CMakeFiles/picongpu-hostonly.dir/build.make CMakeFiles/picongpu-hostonly.dir/build
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
[ 22%] Building CXX object CMakeFiles/picongpu-hostonly.dir/plugins/common/stringHelpers.cpp.o
[ 22%] Building CXX object CMakeFiles/picongpu-hostonly.dir/plugins/common/particlePatches.cpp.o
[ 22%] Building CXX object CMakeFiles/picongpu-hostonly.dir/ArgsParser.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/plugins/common/stringHelpers.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/plugins/common/stringHelpers.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/ArgsParser.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/ArgsParser.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/plugins/common/particlePatches.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/plugins/common/particlePatches.cpp
[ 27%] Building CXX object CMakeFiles/picongpu-hostonly.dir/plugins/hdf5/openPMD/patchReader.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/plugins/hdf5/openPMD/patchReader.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/plugins/hdf5/openPMD/patchReader.cpp
[ 44%] Building CXX object CMakeFiles/picongpu-hostonly.dir/plugins/misc/splitString.cpp.o
[ 44%] Building CXX object CMakeFiles/picongpu-hostonly.dir/plugins/misc/removeSpaces.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/plugins/misc/splitString.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/plugins/misc/splitString.cpp
[ 44%] Building CXX object CMakeFiles/picongpu-hostonly.dir/random/seed/Seed.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/plugins/misc/removeSpaces.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/plugins/misc/removeSpaces.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu-hostonly.dir/random/seed/Seed.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/random/seed/Seed.cpp
[ 50%] Linking CXX static library libpicongpu-hostonly.a
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -P CMakeFiles/picongpu-hostonly.dir/cmake_clean_target.cmake
/ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_link_script CMakeFiles/picongpu-hostonly.dir/link.txt --verbose=ON
/usr/bin/ar qc libpicongpu-hostonly.a  CMakeFiles/picongpu-hostonly.dir/ArgsParser.cpp.o CMakeFiles/picongpu-hostonly.dir/plugins/common/particlePatches.cpp.o CMakeFiles/picongpu-hostonly.dir/plugins/common/stringHelpers.cpp.o CMakeFiles/picongpu-hostonly.dir/plugins/hdf5/openPMD/patchReader.cpp.o CMakeFiles/picongpu-hostonly.dir/plugins/misc/removeSpaces.cpp.o CMakeFiles/picongpu-hostonly.dir/plugins/misc/splitString.cpp.o CMakeFiles/picongpu-hostonly.dir/random/seed/Seed.cpp.o
/usr/bin/ranlib libpicongpu-hostonly.a
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
[ 50%] Built target picongpu-hostonly
make -f CMakeFiles/picongpu.dir/build.make CMakeFiles/picongpu.dir/depend
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
cd /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build && /ccs/home/xxx/lustre/picongpu/src/cmake-3.10.3-Linux-x86_64/bin/cmake -E cmake_depends "Unix Makefiles" /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build /ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu.dir/DependInfo.cmake --color=
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu.dir/DependInfo.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu.dir/depend.internal".
Dependee "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/.build/CMakeFiles/picongpu.dir/depend.internal".
Scanning dependencies of target picongpu
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make -f CMakeFiles/picongpu.dir/build.make CMakeFiles/picongpu.dir/build
make[2]: Entering directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
[ 72%] Building CXX object CMakeFiles/picongpu.dir/main.cpp.o
[ 72%] Building CXX object CMakeFiles/picongpu.dir/versionFormat.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/main.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/main.cpp
[ 77%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/common.cpp.o
[ 77%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/device.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/versionFormat.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/versionFormat.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu[ 77%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/event.cpp.o
/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/common.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/common.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/device.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/device.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/event.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/event.cpp
[ 94%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/memory.cpp.o
[ 94%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/manager/Driver.cpp.o
[ 94%] Building CXX object CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/stream.cpp.o
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/memory.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/memory.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/manager/Driver.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/manager/Driver.cpp
/opt/cray/craype/2.5.13/bin/CC  -DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED -DALPAKA_DEBUG=0 -DBOOST_NO_CXX11_TEMPLATE_ALIASES -DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER_ID=GNU -DCMAKE_CXX_COMPILER_VERSION=4.9.3 -DCMAKE_SYSTEM=Linux-3.0.101-0.47.106.29-default -DCMAKE_SYSTEM_PROCESSOR=x86_64 -DCMAKE_VERSION=3.10.3 -DCUPLA_STREAM_ASYNC_ENABLED=1 -DENABLE_HDF5=1 -DPIC_ENABLE_PNG=1 -DPIC_VERBOSE_LVL=1 -DPMACC_VERBOSE_LVL=0 -DUSE_FREETYPE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -I/ccs/home/xxx/lustre/picongpu/inputData/myLWFA-test-pngwriter-fail/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/pmacc/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/.. -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/alpaka/include -I/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/boost-1.65.1/install-dir/include -isystem /usr/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/splash/install-dir/include -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/freetype-2.9.1/install-dir/include/freetype2 -isystem /lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include  -fopenmp -fopenmp -Wall -Wno-unknown-pragmas -Wextra -Wno-unused-parameter -Wno-unused-local-typedefs -O3 -DNDEBUG   -fopenmp -std=c++11 -ftemplate-depth-512 -Wall -Wextra -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-local-typedefs -Wno-attributes -Wno-reorder -Wno-sign-compare -std=c++11 -o CMakeFiles/picongpu.dir/lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/stream.cpp.o -c /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/thirdParty/cupla/src/stream.cpp
In file included from /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/versionFormat.cpp:39:0:
/lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include/pngwriter.h:119:25: error: 'FT_Bitmap' has not been declared
void my_draw_bitmap( FT_Bitmap * bitmap, int x, int y, double red, double green, double blue);
^
/lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include/pngwriter.h:120:31: error: 'FT_Bitmap' has not been declared
void my_draw_bitmap_blend( FT_Bitmap * bitmap, int x, int y,double opacity,  double red, double green, double blue);
^
make[2]: *** [CMakeFiles/picongpu.dir/versionFormat.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from /lustre/atlas2/venxxx/scratch/xxx/picongpu/src/picongpu-src/include/picongpu/plugins/output/images/PngCreator.tpp:38:0,
from /lustre/atlas2/venxxx/scratch/xxx/picongpu/src/picongpu-src/include/picongpu/plugins/output/images/PngCreator.hpp:127,
from /lustre/atlas2/venxxx/scratch/xxx/picongpu/src/picongpu-src/include/picongpu/plugins/PluginController.hpp:33,
from /lustre/atlas2/venxxx/scratch/xxx/picongpu/src/picongpu-src/include/picongpu/unitless/starter.unitless:23,
from /lustre/atlas2/venxxx/scratch/xxx/picongpu/src/picongpu-src/include/picongpu/simulation_defines.hpp:53,
from /lustre/atlas/scratch/xxx/venxxx/picongpu/src/picongpu-src/include/picongpu/main.cpp:24:
/lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include/pngwriter.h:119:25: error: 'FT_Bitmap' has not been declared
void my_draw_bitmap( FT_Bitmap * bitmap, int x, int y, double red, double green, double blue);
^
/lustre/atlas/scratch/xxx/venxxx/picongpu/src/pngwriter/install-dir/include/pngwriter.h:120:31: error: 'FT_Bitmap' has not been declared
void my_draw_bitmap_blend( FT_Bitmap * bitmap, int x, int y,double opacity,  double red, double green, double blue);
^
make[2]: *** [CMakeFiles/picongpu.dir/main.cpp.o] Error 1
make[2]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make[1]: *** [CMakeFiles/picongpu.dir/all] Error 2
make[1]: Leaving directory `/lustre/atlas2/venxxx/scratch/xxx/picongpu/inputData/myLWFA-test-pngwriter-fail/.build'
make: *** [all] Error 2

ERROR: Could not successfully run make install in build directory:
.build
```

--- a/src/pngwriter.h
+++ b/src/pngwriter.h
@@ -65,6 +65,7 @@
 #ifndef NO_FREETYPE
 #include <ft2build.h>
 #include FT_FREETYPE_H
+#include FT_IMAGE_H
 #endif
 
 #include <iostream>

